### PR TITLE
useAuctions Refactor / Hide defective listings / Optional TLS

### PR DESCRIPTION
**Pull request by kespinola**
### Changes
- Refactor use infinite scroll listings into 2 hooks. 1 to group listings another to handle loading more info with infinite scroll.
- Optional TLS
- Hides defective listings

### CODE REVIEW

File: js/packages/web/src/actions/getStorefront.ts [hunk 1]
```diff
@@ -108,7 +109,7 @@ export const getStorefront = async (
   const client = createClient({
     url: REDIS_URL,
     socket: {
-      tls: true,
+      tls: REDIS_TLS_ENABLED,
```
Comment (sjc5) [inline js/packages/web/src/actions/getStorefront.ts:112]: I think we'll need to spread an empty object in the event of local dev and an object with `tls: true` for prod (false isn't a valid value for tls here I guess)

Comment (sjc5) [reply js/packages/web/src/actions/getStorefront.ts:112]: Actually `tls: true` is just required for the build step to complete it seems :-/

**Comment by sjc5** [comment]
@kespinola I can't request your review on this PR, but can you please make sure you're OK with these changes: https://github.com/holaplex/metaplex/pull/244/commits/1b6f9d24f40e2730f1bad57238b1c07c5b7f346a ?

**Comment by kespinola** [comment]
@sjc5 solid work. Tweaked some items in this commit. +1

https://github.com/holaplex/metaplex/pull/244/commits/0df6e2d396ec4cd51c49ae568953fb208358d9eb

**Pull request merged**